ui/src/middlewares/__tests__/clearStateDispatcher.test.js

```diff
@@ -0,0 +1,111 @@
+import { LOCATION_CHANGE } from 'react-router-redux';
+
+import { CLEAR_STATE } from '../../actions/actionTypes';
+import middleware from '../clearStateDispatcher';
+
+describe('clearStateDispatcher middleware', () => {
+  describe('on LOCATION_CHANGE returns next(original action) and', () => {
```

Comment:
two ``describe``s?  :)

Reply:
there is nothing wrong about that, `describe` can be used to organize/group test cases. Nested `describe` are not new to the codebase.

This is done like this, when we have another case where we dispatch `CLEAR_STATE`, we'll add another `describe('on ANOTHER_THING return...`